### DISCUSSION TITLE
如何给选定的一组文献批量添加tag

### DISCUSSION THREAD

**Discussion by silvewheat** [windingwind/zotero-actions-tags#212] [Q&A] [answered]
有没有办法给选中的多篇文献批量添加tag？

**Comment by windingwind** [comment, marked as answer]
<img width="351" alt="image" src="https://github.com/windingwind/zotero-actions-tags/assets/33902321/f7ee093c-a744-4d67-8122-a5192a277a33">

**Comment by silvewheat** [comment]
感谢解答！
不过这样是不是就得给每一个想添加的tag单独创建一个快捷键或添加到右键菜单？

**Reply by silvewheat** [reply]
虽然理想中最好是能在右键菜单中直接创建新的tag并添加，但这样已经很好用了，谢谢！

**Reply by windingwind** [reply]
要批量添加已有标签到文献，选中多个文献条目后，拖拽到左下角标签管理器的标签上。

**Reply by silvewheat** [reply]
哦，原来还可以这样！感谢！

**Reply by sjhsbhqf** [reply]
> 要批量添加已有标签到文献，选中多个文献条目后，拖拽到左下角标签管理器的标签上。

现在这个版本是不能用了吗？往标签上拖不起作用

tag插件的版本号：2.0.5
zotero的版本号：7.0
